<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>828</width>
    <height>470</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GitUI Main Window</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="1" column="2">
     <widget class="QTextBrowser" name="wyswietlacz">
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:36pt; font-weight:600;&quot;&gt;Welcome to the&lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:36pt; font-weight:600;&quot;&gt;GitUI&lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:24pt;&quot;&gt;Select repository from &lt;/span&gt;&lt;span style=&quot; font-size:24pt; font-style:italic;&quot;&gt;Repository-&amp;gt;Open&lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:24pt;&quot;&gt;and enjoy.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QPushButton" name="pushButtonStatus">
        <property name="text">
         <string>Status</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="pushButtonCommit">
        <property name="text">
         <string>Commit</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="pushButtonLog">
        <property name="text">
         <string>Log</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="labelGitui">
      <property name="font">
       <font>
        <pointsize>26</pointsize>
       </font>
      </property>
      <property name="text">
       <string>GitUI</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="labelGit">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>828</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuRepository">
    <property name="title">
     <string>Repository</string>
    </property>
    <addaction name="actionRepositoryOpen"/>
    <addaction name="actionRepositoryCreate"/>
    <addaction name="actionRepositoryDelete"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionFileAdd"/>
   </widget>
   <widget class="QMenu" name="menuBranch">
    <property name="title">
     <string>Branch</string>
    </property>
    <addaction name="actionBranchBranch"/>
   </widget>
   <widget class="QMenu" name="menuRemote">
    <property name="title">
     <string>Remote</string>
    </property>
    <addaction name="actionModify_Repositories"/>
    <addaction name="actionRemoteClone"/>
    <addaction name="actionRemotePush"/>
    <addaction name="actionRemoteFetch_2"/>
   </widget>
   <widget class="QMenu" name="menuUser">
    <property name="title">
     <string>User</string>
    </property>
    <addaction name="actionUserSettings"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionIndex"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuRepository"/>
   <addaction name="menuFile"/>
   <addaction name="menuBranch"/>
   <addaction name="menuRemote"/>
   <addaction name="menuUser"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionRepositoryOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionRepositoryCreate">
   <property name="text">
    <string>Create</string>
   </property>
  </action>
  <action name="actionRepositoryDelete">
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="actionAdd">
   <property name="text">
    <string>Add</string>
   </property>
  </action>
  <action name="actionFileAdd">
   <property name="text">
    <string>Add</string>
   </property>
  </action>
  <action name="actionFileDelete">
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="actionBranchBranch">
   <property name="text">
    <string>Branch / Merge</string>
   </property>
  </action>
  <action name="actionBranchCheckout">
   <property name="text">
    <string>Checkout</string>
   </property>
  </action>
  <action name="actionRemoteClone">
   <property name="text">
    <string>Clone</string>
   </property>
  </action>
  <action name="actionRemotePush">
   <property name="text">
    <string>Push/Pull</string>
   </property>
   <property name="iconText">
    <string>Push&amp;Pull</string>
   </property>
  </action>
  <action name="actionRemoteFetch_2">
   <property name="text">
    <string>Fetch</string>
   </property>
  </action>
  <action name="actionRemoteFetch">
   <property name="text">
    <string>Fetch</string>
   </property>
  </action>
  <action name="actionUserSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionModify_Repositories">
   <property name="text">
    <string>Modify Tracked Repositories</string>
   </property>
  </action>
  <action name="actionIndex">
   <property name="text">
    <string>Index</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
